content/en/docs/how-tos/creating-a-pipeline.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Creating a Pipeline"
+description: How to use existing functionalities to create a simple 2-stage pipeline
+---
+
+Prow does not support pipelines. Hence, when submitting a new pull request, all tests are executed, including resource-intensive tests that can lead to increased infrastructure costs. To address this limitation, OpenShift CI introduces an abstraction layer utilizing GitHub's branch protection rules. This enables the use of both manual (triggered by a user) and automatic (triggered by a bot) two-stage pipelines.
+
+## Marking CI job as second stage job
+
+To enable two-stage pipelines, two actions are required. First, a specific branch protection setting must be activated within the Prow configuration of the repository (or organization).
+
+{{< alert title="Note" color="info" >}}
+Branch protection configuration could be also enabled on branch level. It will however disable possibility of using automatic pipelines as they work on repository and organization level.
```

Comment:
[](url)
```suggestion
Branch protection configuration can also be enabled on the branch level. It will, however, disable the possibility of using automatic pipelines as they work on the repository and organization level.
```